widget/button.go

```diff
@@ -268,12 +268,12 @@ func (o ButtonOptions) TextAndImage(label string, face text.Face, image *ButtonI
 			)
 			c.AddChild(b.text)
 
-			b.graphic = NewGraphic(
+			b.Graphic = NewGraphic(
```

Comment:
Why'd ya expose Graphic here?

Reply:
ups sorry I had local changes gonna roll this back